src/com/google/javascript/jscomp/CompilerOptions.java

```diff
@@ -2411,6 +2423,34 @@ public static LanguageMode fromString(String value) {
       }
       return null;
     }
+
+    public int toInteger() {
```

Comment:
I can remove all of this for now. Originally the intent was that this was a much more flexible way throughout the compiler to test for specific code levels rather than continuing to add functions like `es5orHigher`.